ros_ign_bridge/test/serverclient/ign_server.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <ignition/msgs/boolean.pb.h>
+#include <ignition/msgs/world_control.pb.h>
+
+#include <ignition/transport.hh>
+
+#include <atomic>
+#include <chrono>
+#include <csignal>
+#include <iostream>
+#include <string>
```

Comment:
unused?

Reply:
Fixed in https://github.com/ignitionrobotics/ros_ign/pull/211/commits/b2a37daabf00db7c9bfed437d7d096dc535f597a